docs/modules/consul.md

```diff
@@ -2,6 +2,8 @@
 
 Testcontainers module for [Consul](https://github.com/hashicorp/consul). Consul is a tool for managing key value properties. More information on Consul [here](https://www.consul.io/).
 
+Since: Not released yet
```

Comment:
any idea how to "automate" it? I am sure we will be forgetting to update it almost every release (happened already with the `latest_version` in `mkdocs.yml`). Or... does mkdocs support conditioning?

Reply:
In `mkdocs.yml`, having

```yml
extra:
    redpanda_version: 1.17.3
```

we can do the following:

```
{% if extra.redpanda_version %}
Since: {{extra.redpanda_version}}
{% endif %}
```

Reply:
For automation and specific things, I've been looking at https://github.com/actions/github-script but I think that may we will need something more complex in the future and maybe create our own `testcontainers-bot` having all the logic in our own backed.